linux/ZramMeter.c

```diff
@@ -4,7 +4,7 @@
 #include "Meter.h"
 #include "Platform.h"
 
-static int ZramMeter_attributes[] = {
+const static int ZramMeter_attributes[] = {
```

Comment:
`static const`: Storage class comes before type modifiers.

Reply:
it's already fixed.